sample-compose/app/build.gradle.kts

```diff
@@ -101,6 +109,8 @@ android {
                 }
             }
         }
+        // Disable device's animation for instrument testing
+        // animationsDisabled = true
```

Comment:
Is it common for us to disable animations for UI testing? I think animations should always be enabled, so we can be sure there are no animation-related bugs in the app 💭 (a.k.a I think we should remove the comment)